MIGRATION.md

```diff
@@ -807,7 +1101,7 @@ We also introduced API changes for the checkbox. This is an overview of the rena
 - `disabled` was renamed to `isDisabled`;
 - `indeterminate` was renamed to `isIndeterminate`.
 
-#### How to migrate your checkbox components
+##### How to migrate your checkbox components
```

Comment:
For consistency 'checkbox' should be capitalised.

Reply:
```suggestion
##### How to migrate your Checkbox components
```